wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/InmemoryWorkspaceActivityDao.java

```diff
@@ -100,11 +100,8 @@ public void setStatusChangeTime(String workspaceId, WorkspaceStatus status, long
                 a -> {
```

Comment:
This and the above filter could really be simplified to `a -> a.getStatus() == status && isGreater(a.getLastStopped(), timestamp)`

Reply:
I've just applied IDE suggestion, but you're right and now I've applied your suggestion )